02_configure_host.sh

```diff
@@ -26,6 +26,9 @@ ANSIBLE_FORCE_COLOR=true ansible-playbook \
 # Allow local non-root-user access to libvirt
 sudo usermod -a -G "libvirt" $USER
 
+# Restart libvirtd service to get the new group membership loaded
```

Comment:
Should we perhaps do this (and the usermod above) conditionally, so we don't restart libvirt unneccesarily?

E.g something like:

```
  if ! id $USER | grep -q libvirt; then
    sudo usermod -a -G "libvirt" $USER
    sudo systemctl restart libvirtd
fi
```